eventstore/store_total_costs.go

```diff
@@ -0,0 +1,40 @@
+package eventstore
+
+import (
+	"encoding/json"
+
+	"github.com/alphagov/paas-billing/eventio"
+)
+
+var _ eventio.TotalCostReader = &EventStore{}
+
+func (s *EventStore) GetTotalCost() ([]eventio.TotalCost, error) {
+	tx, err := s.db.Begin()
+	if err != nil {
+		return nil, err
+	}
+
+	defer tx.Rollback()
```

Comment:
Can we do this without the transaction? I don't think it makes any difference, but the code would be a bit simpler without the unnecessary rollback. (Unnecessary since we're only running one `select`)